model/device.go

```diff
@@ -93,6 +93,136 @@ const (
 	DevTypeTest       = "Test"
 )
 
+// Consts containing defaults for all device configs.
+const (
+	defaultActPeriod int64 = 60
+	defaultMonPeriod int64 = 60
+)
+
+// Consts containing defaults for a controller config.
+const (
+	defaultControllerInputs  string = "A0,X50,X51,X60,X10"
+	defaultControllerOutputs string = "D14,D15,D16" // Peripheral Power.
+)
+
+// controllerDefaultVars is a map which relates controller variable names to their default values
+var controllerDefaultVars = map[string]string{
+	"AlarmNetwork":         "10",
+	"AlarmPeriod":          "5",     // 5 seconds.
+	"AlarmRecoveryVoltage": "840",   // 25.4545 V.
+	"AlarmVoltage":         "825",   // 25 V.
+	"AutoRestart":          "600",   // 10 minutes.
+	"Power":                "false", // initialise to OFF.
+	"Pulses":               "3",     // 3 pulses per cycle.
+	"PulseWidth":           "2",     // 1s ON, 1s OFF.
+	"PulseCycle":           "30",    // 30s between flashes.
+	"Suppress":             "false",
```

Comment:
Suppress is an obsolete var too, it's PulseSuppress. The defaults are in the config guide: https://docs.google.com/document/d/1xdS0qk25hFhkAivB2wlzd5U0ZxGEyk_avg-NGPYXtc4/edit